src/main/java/com/hillel/items_exchange/controller/UserController.java

```diff
@@ -0,0 +1,37 @@
+package com.hillel.items_exchange.controller;
+
+import com.hillel.items_exchange.dto.UserDto;
+import com.hillel.items_exchange.model.User;
+import com.hillel.items_exchange.service.UserService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.security.Principal;
+
+@RestController
+@RequestMapping("/user")
+@CrossOrigin(origins = "http://localhost:4200")
+@RequiredArgsConstructor
+@Slf4j
```

Comment:
Why did you added the logger here and didn't use it?

Reply:
@21November the question remains. :arrow_up:

Reply:
At one point, the logger was used. Then the code was changed and the logger was no longer needed. I'm gonna delete this annotation.

Reply:
line 41 of this code uses logger. 

log.warn(e.getMessage(), e);